packages/design-system/src/styles/settings/mixins/_high-contrast-mode-label.scss

```diff
@@ -0,0 +1,31 @@
+@mixin high-contrast-mode-label-add($label-en, $label-es: null) {
```

Comment:
How will the Spanish translation get added in? I see that there's a parameter for it, but like who is in charge of providing those?

Reply:
That's a good question. Maybe we should speak with a content person? I was hesitant to add the text in, as I don't speak Spanish (also thinking regional considerations?).